infra/general/KCryptoWrapper.class.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+class KCryptoWrapper
+{
+    public static function getEncryptor()
+    {
+	if (extension_loaded('mcrypt')){
+	    return new McryptWrapper();
+	}else{
+	    return new OpenSSLWrapper();
+	}
+    }
+
+}
+
+class OpenSSLWrapper
+{
+    const AES_BLOCK_SIZE = 16;
+    const DES3_BLOCK_SIZE = 8;
+    const AES_METHOD = 'AES-128-CBC';
+    const DES3_METHOD = 'DES-EDE3';
+
+    public static function random_pseudo_bytes($bytes)
```

Comment:
rename bytes to length or size, since it may be confusing.

Reply:
I don't think it's confusing but renamed it to $length anyhow since it's not worth arguing about.